src/Razor/test/Microsoft.AspNetCore.Razor.Test.Common.Tooling/Formatting_NetFx/HtmlFormatting.cs

```diff
@@ -0,0 +1,108 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Razor.PooledObjects;
+using Microsoft.AspNetCore.Razor.Test.Common.Mef;
+using Microsoft.CodeAnalysis.Razor.Logging;
+using Microsoft.CodeAnalysis.Text;
+using Microsoft.VisualStudio.LanguageServer.Protocol;
+using Microsoft.VisualStudio.Text;
+using Microsoft.VisualStudio.Utilities;
+using Microsoft.WebTools.Languages.Shared.ContentTypes;
+
+namespace Microsoft.CodeAnalysis.Razor.Formatting;
+
+internal static class HtmlFormatting
+{
+    public static Task<TextEdit[]?> GetDocumentFormattingEditsAsync(ILoggerFactory loggerFactory, Uri uri, string generatedHtml, bool insertSpaces, int tabSize)
```

Comment:
Is there any doc that needs to be updated to reflect that this method was moved over? even though the class is internal, I personally still like it if more of our methods had triple slash doc comments explaining them. what do you think?

Reply:
This is kinda funny, because I seem to remember talking to you about code comments in the past, and I was on the side of more comments, and you were arguing for less  😛

To your first question about docs, no. This is purely internal test code. It is called in one place in main, and this PR expands that to two. I wouldn't expect there to be any others added any time soon.
Regarding doc comments, I have nothing against them in general, but in this case I don't honestly know what they would say that is not communicated by the class and method name. `HtmlFormatting.GetDocumentFormattingEditsAsync` with a comment that says "Gets document formatting edits for Html" doesn't add much value IMO. Was there something specific you think would be worth mentioning in a doc comment?